### DISCUSSION TITLE
Invio e recezione messaggi di posta elettronica

### DISCUSSION THREAD

**Discussion by ghost** [iisgiua/giuaschool#571] [Richieste di aiuto]
Buongiorno, vorrei un aiuto.

Dal registro installato su Hosting di Altervista, sto riscontrando seri problemi che mi stanno mettendo in difficoltà con la mia scuola. Le e-mail sia di prova, sia per inviare le credenziali, non arrivano ai docenti. Ho inserito la funzione di inviare le e-mail con il PHP ma continua a non funzionare, oltre che il recovery password è inaccessibile senza questa funzione. Ho inserito l'indirizzo della segreteria per inviare le e-mail, come 'mittente delle e-mail' sui i parametri della scuola, ma anche lì non funziona.

Come posso risolvere? Chiedo una mano a i professionisti che ho visto che si intendono molto in questo forum di supporto.

@trinko, @bfabio, @zerai, @MatteoGheza.  Scusate il ping e il disturbo, ma mi serve urgentemente!

**Comment by trinko** [comment]
Quando fai il test nella pagina di configurazione dell'email viene visualizzato un messaggio di errore? Quale?

**Reply by ghost** [reply]
Non viene visualizzato nessun messaggio di errore. Non arriva neanche l'e-mail, nonostante la configurazione.

**Comment by ghost** [comment]
Non viene visualizzato nessun messaggio di errore. Non arriva neanche l'e-mail, nonostante la configurazione.

@trinko

**Comment by trinko** [comment]
Carica il log degli errori seguendo la procedura seguente:
- come utente amministratore, andare alla pagina **Sistema > Manutenzione** e abilitare il debug
- andare alla pagina di configurazione email ed inviare la mail di test
- tornare alla pagina **Sistema > Manutenzione** e estrarre il log (indicando l'ora di esecuzione del punto precedente)
- caricare qui il log, in modo che lo possa esaminare: NON FARE COPIA E INCOLLA, ma allegare il file 
- uscire dalla modalità debug

**Comment by ghost** [comment]
Ecco a lei! 

@trinko 

[2023-09-29_21-10.log](https://github.com/iisgiua/giuaschool/files/12770843/2023-09-29_21-10.log)

**Comment by trinko** [comment]
Il log non riporta nessun errore: la procedura di invio viene quindi eseguita senza errori.
Se sei sicuro che la mail non arrivi (hai controllato che non venga identifcata come SPAM?) allora è un problema di Altervista. 

Hai impostato come email di invio delle notifiche (**SCUOLA > ISTITUTO > Indirizzo email del mittente delle notifiche inviate dal registro**) 
la stessa email che è configurata per l'utente del sito Altervista? Se non l'hai fatto, prova.

**Comment by ghost** [comment]
Aspetti, provo un attimo e le faccio sapere.

@trinko

**Comment by ghost** [comment]
Non funziona, non arriva mai.

@trinko

**Comment by ghost** [comment]
So che c'è un esperto in particolare. @manuelbarberi potrebbe aiutarci?

**Reply by manuelbarberi** [reply]
Mi scriva sulla mail: manuelbarberi.mb@libero.it
